model-integration/src/test/java/ai/vespa/rankingexpression/importer/tensorflow/SoftmaxImportTestCase.java

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package ai.vespa.rankingexpression.importer.tensorflow;
+
+import ai.vespa.rankingexpression.importer.ImportedModel;
+import ai.vespa.rankingexpression.importer.configmodelview.ImportedMlFunction;
+import org.junit.Assert;
+import org.junit.Test;
+
+
+import static org.junit.Assert.assertNotNull;
+
+/**
+ * @author lesters
+ */
+public class SoftmaxImportTestCase {
+
+    @Test
+    public void testSoftmaxImport() {
+        TestableTensorFlowModel model = new TestableTensorFlowModel("test", "src/test/models/tensorflow/softmax/saved", 1, 5);
+        ImportedModel.Signature signature = model.get().signature("serving_default");
+        Assert.assertEquals("Has skipped outputs",
```

Comment:
Since I'm already nitpicking: This should say "Has no skipped outputs" (the message describes what you're asserting).

Reply:
This doesn't really make sense to me. This message is output when the assertion fails, and should help in figuring out what the problem is. In this case "has no skipped outputs" would just be confusing. Perhaps something more descriptive, e.g. "Should have no skipped outputs" ?

Reply:
It is an assertion, i.e something declarative. When it fails you'll see
java.lang.AssertionError: "Description of the assertion"

More importantly, we should optimize for making the test code easy to read, which means describing the assertions rather than writing "error messages which will be output when they fail":
`assert...(description of what we are asserting, ...
`not
`assert...(error message, ...`


Anyway, this isn't that important ...